src/store/market/market-state.ts

```diff
@@ -0,0 +1,22 @@
+import { MarketCoinsSortFieldEnum } from '../../enums/market-coins-sort-field.enum';
+import { LoadableEntityState } from '../types';
+import { createEntity } from './../create-entity';
+import { MarketCoin } from './market.interfaces';
+
+export interface MarketState {
+  favouriteTokens: Array<string>;
+  sortField: MarketCoinsSortFieldEnum;
+  tokens: LoadableEntityState<Array<MarketCoin>>;
+  tokensSlugs: LoadableEntityState<Record<string, string>>;
```

Comment:
what `tokensSlugs` are these?  rename to understand what does this record represents